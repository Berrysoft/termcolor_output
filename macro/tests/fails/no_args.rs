use termcolor_output as tco;

fn main() {
    tco::write!();
}
