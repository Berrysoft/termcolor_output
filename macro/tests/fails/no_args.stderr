error: colored! macro can't be called without arguments
 --> $DIR/no_args.rs:4:5
  |
4 |     colored!();
  |     ^^^^^^^^^^^
  |
  = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
